web_config.py

```
@@ -0,0 +1,848 @@
+            stripped = line.strip()
+            if stripped == "" or stripped.startswith("#") or stripped.startswith(";"):
+                pending.append(line)
+            elif "=" in stripped:
+                key = stripped.split("=", 1)[0].strip()
+                if key in new_pairs:
+                    result.extend(pending)
+                    pending = []
+                    result.append(f"{key} = {new_pairs[key]}\n")
+                    written.add(key)
+                else:
+                    # Key was deleted – discard its associated comments too
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)